### ISSUE TITLE
set useStylelintConfigOverrides true is not work?

### ISSUE THREAD

**Issue by abrcdf1023** [mrmlnc/vscode-stylefmt#11]
Hello i have some config setting issue

This's my vscode settings.json

```
"stylelint.enable": true,
"stylelint.config": {
    "rules": { ... }
 }
"stylefmt.autoFormatOnSave": true,
"stylelint.configOverrides": {
     "indentation": "tab"
 },
"stylefmt.useStylelintConfigOverrides": true
```

But it no work , it's format as default setting.

Can it have a feature that to config vscode-setting like this?

`
"stylefmt.config":{
    "rules":{
        "indentation": "tab",
        ...
    }
}
`

If so , that will be awesome !!

Btw support rules are here

https://github.com/morishitter/stylefmt#stylelint-rules-that-stylefmt-can-handle

**Comment by mrmlnc** [comment]
Hello, @abrcdf1023,

Sorry for delay, just work :construction_worker:.

Now the extension gets the settings only from `.stylelintrc` file and `stylelint.configOverrides` options.

**stylelintrc**

``` json
{
  "rules": {
    "color-hex-case": "lower"
  }
}
```

**settings.json**

``` json
{
  "stylelint.configOverrides": {
    "color-hex-case": "upper"
  },
  "stylefmt.useStylelintConfigOverrides": true
}
```

You want to configure the extension using `settings.json` with `stylefmt.config` option?

**Comment by abrcdf1023** [comment]
Yes, but if configoverride is work that will not important anymore.

Thanks for your hard work!
Have a nice day.

**Comment by mrmlnc** [comment]
@abrcdf1023, I can close this issue?

**Comment by abrcdf1023** [comment]
Yes, you can.

**Comment by abrcdf1023** [comment]
@mrmlnc 

Sorry to bother you all the times

I found that if my .stylelintrc not in root but in stylesheet folder it will not be loaded, it's predictable?

![default](https://cloud.githubusercontent.com/assets/12894459/18153140/1f65a068-702e-11e6-8919-9272467f8d71.png)

settings.json not work as predictable also.

![2](https://cloud.githubusercontent.com/assets/12894459/18153179/5ecac12a-702e-11e6-9c80-671aaea833da.png)

My OS is win10.

Maybe I should buy a mac...

**Comment by mrmlnc** [comment]
@abrcdf1023,

You're right, `.stylelintrc` used only from the root directory. I'll start working on this soon.

**Comment by mrmlnc** [comment]
Fixed in version **1.2.1**. Please, try new version.
